test/github-release.ts

```diff
@@ -41,7 +41,7 @@ describe('GitHubRelease', () => {
         .get('/repos/googleapis/foo')
         .reply(200, repoInfo)
         .get(
-          '/repos/googleapis/foo/pulls?state=closed&per_page=100&sort=merged_at&direction=desc'
+          '/repos/googleapis/foo/pulls?state=closed&per_page=25&sort=merged_at&direction=desc'
```

Comment:
The magic limit that appears to be here makes me nervous.  Should it?

Reply:
Left a comment about why `25` was chosen.